fs-utils.go

```diff
@@ -21,45 +21,36 @@ import (
 	"unicode/utf8"
 )
 
-// validBucket regexp.
-var validBucket = regexp.MustCompile(`^[a-z0-9][a-z0-9\.\-]{1,61}[a-z0-9]$`)
+// validVolname regexp.
+var validVolname = regexp.MustCompile(`^.{3,63}$`)
```

Comment:
> I think we need to have more validation here ie name with /, '\' and other special chars are also getting passed. I see Windows doesn't allow / \ : * ? " < > | and linux doesn't allow /

Yes we need those for platform specific, what i thought was Makedir() would return errors in such cases. Need to see what kind of errors it does. 

But its a good idea to validate it before even attempting Makedir()